win/pkgs/grpc.ps1

```diff
@@ -68,6 +68,25 @@ cmd /c xcopy /i /f /y "pdb\*.pdb" "${Env:ProgramFiles(x86)}\grpc\bin"
 Get-ChildItem "${Env:ProgramFiles(x86)}/grpc" -Filter *.dll -Recurse | Foreach-Object { New-Item -Force -ItemType SymbolicLink -Path "${Env:SystemRoot}\System32\$_" -Value $_.FullName }
 
 popd
+
+# Known issue:
+#   - abseil header is not installed via grpc
+#     https://github.com/grpc/grpc/issues/25949
```

Comment:
The issue has been closed.
Is the issue still there?

Reply:
Yes. This is closed as by design and the maintainer is suggesting to manually install the headers.

Reply:
That make sense